docs/conf.py

```diff
@@ -12,28 +12,76 @@
 #
 import os
 import sys
+import shutil
+import subprocess
+
 import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath('../lib'))
-sys.path.insert(0, os.path.abspath('../lib/sedna'))
+try:
+    import m2r2
+except ModuleNotFoundError:
+    subprocess.check_call([sys.executable, "-m", "pip", "install", "m2r2"])
+
+try:
+    import autoapi
+except ModuleNotFoundError:
```

Comment:
name `ModuleNotFoundError` is not defined

Reply:
`ModuleNotFoundError` is only available in Python 3.6 and later versions, same with `sedna`